python/torch_mlir/csrc/reference_lazy_backend/backend_impl.cpp

```diff
@@ -106,7 +109,15 @@ class ReferenceLazyBackendImpl : public torch::lazy::TorchMlirBackendImpl {
         at::Tensor tensor = mlir_data->mlir_info()->tensor;
         stack.emplace_back(tensor);
       }
+
+      // count number of inputs
+      auto name = mlir_data->mlir_info()->name;
+      if (startswith(name, "input_")) {
```

Comment:
does this name end up in the graph or any other data structure we can see without doing this? It feels really hacky to print to stdout like this, even for testing.

Reply:
No it does not. Plus, this is the reference backend and thus not intended to be used anywhere outside of testing and only as a reference for how a real backend subclass should be implemented.

Reply:
okay. can you make this name more verbose like "only_for_testing_print_this_"

Reply:
Is that strictly necessary? I think it's fairly obvious that the printing out is not a necessary step in implementing a lazy backend.

Reply:
I believe so. Otherwise it can be confusing -- is it leftover debugging code? something else?

Reply:
okay, I'll add a comment explaining the purpose